src/lib/temple/back/evm-validation-schemas/index.ts

```diff
@@ -48,6 +49,37 @@ export const ethPersonalSignPayloadValidationSchema = mixedSchema<
   return false;
 }).required();
 
+export const addEthAssetPayloadValidationSchema = objectSchema()
+  .shape({
+    type: stringSchema().required(),
+    options: objectSchema()
+      .shape({
+        address: evmAddressValidationSchema().required()
+      })
+      .required()
+  })
+  .required();
+
+export const addEthChainPayloadValidationSchema = tupleSchema([
+  objectSchema()
+    .shape({
+      chainId: stringSchema().required(),
+      chainName: stringSchema().required(),
+      nativeCurrency: objectSchema()
+        .shape({
+          name: stringSchema().required(),
+          symbol: stringSchema().required(),
+          decimals: numberSchema().integer().positive().required()
```

Comment:
Please check how it works with 0 decimals. If everything is fine, just resolve this conversation

Reply:
got validation error as expected